packages/api/cms-api/src/common/filter/mikro-orm.ts

```diff
@@ -212,3 +251,23 @@ export function searchToMikroOrmQuery(search: string, fields: string[]): ObjectQ
         $and: ands,
     };
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function mikroOrmQuery({ search, filter }: { search?: string; filter?: any }, repository: EntityRepository<any>): ObjectQuery<any> {
```

Comment:
We need e better name for this:

- argsToMikroOrmQuery
- createMikroOrmQuery

Reply:
- argsToMikroOrmQuery: args is very unspecific
- createMikroOrmQuery: just adding ad create prefix doesn't improve the name much

Maybe add something with filter (search is also a kind of filter) and/or gql?

- filterToMikroOrmQuery
- gqlArgsToMikroOrmQuery
- gqlToMikroOrmQuery
- gqlFilterToMikroOrmQuery

Reply:
renamed in [5d5601f](https://github.com/vivid-planet/comet/pull/2238/commits/5d5601f441813b1f741f76fdd2835e02ed8fd0a2)